### PR TITLE
Add severity to incident

### PR DESCRIPTION
**Pull request by ereteog**
> Related #https://github.com/advthreat/iroh/issues/5684

This PR adds `severity` field to incident.
It also introduces the `Critical` value for `HighMedLow`.

### CODE REVIEW

File: src/ctim/schemas/incident.cljc [hunk 1]
```diff
@@ -61,7 +61,9 @@
    (f/entry :assignees [c/ShortString]
             :description "a set of owners assigned to this incident")
    (f/entry :promotion_method v/PromotionMethod
-            :description "identifies how the incident was promoted")))
+            :description "identifies how the incident was promoted")
+   (f/entry :severity v/HighMedLow
+            :description "specifies the serverity level of an Incident")))
```
Comment (frenchy64) [inline src/ctim/schemas/incident.cljc:66]: `serverity` => `severity`

**Review by frenchy64 — APPROVED** [review]
Spotted a typo (don't forget to regen docs), otherwise looks good!

File: src/ctim/schemas/incident.cljc (hunk 1 above)
Comment (ereteog) [reply src/ctim/schemas/incident.cljc:66]: it's a new word to define a capacity to serve :-)

Comment (frenchy64) [reply src/ctim/schemas/incident.cljc]: Critical capacity to serve xD

**Comment by gbuisson** [comment]
@ereteog I think we should consider renaming the HighMedLow schema or duplicating it.

**Pull request merged**